A43-sdk-authorization.md

```diff
@@ -0,0 +1,364 @@
+A43: SDK authorization
+
+Author(s): [Ashitha Santhosh](https://github.com/ashithasantosh)
+Approver: 
+Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+Implemented in: <language, ...>
+Last updated: 2021-06-17
+Discussion at: (filled after thread exists)
+
+##Abstract
+
+gRPC is currently implementing an open-source standard authorization solution
+based on Envoy RBAC. This proposal discusses how this framework would be made 
+available to OSS gRPC users to perform per-RPC authorization checks.
+
+##Background
```

Comment:
Sorry for giving the confusing advice during our 1:1. But the current content in `Background` is closer to `Implementation details`. In `Background`, I suggest discussing 1) the difference between authentication vs authorization, 2) client-side authz (i.e., hostname verification) vs server-side authz, and 3) mention xDS authz.

Reply:
No worries :) Updated the background.